content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1007 @@
+
+```rust
+// Despawns all descendants of an entity (its children, its childrens' children, etc)
+commands.entity(e).despawn_descendants();
+
+// Removes the given children from the entity
+commands.entity(parent).remove_children(&[child1, child2]);
+```
+
+## UI
+
+### Overflow::Hidden
+
+<div class="release-feature-authors">authors: @Davier</div>
+
+UI now repects the flexbox `Overflow::Hidden` property. This can be used to cut off child content, which is useful when building things like scrollable lists:
+
+![overflow hidden](overflow.png)
+
+### Text2D Transforms
+
+<div class="release-feature-authors">authors: @nside</div>
+
+`Text2d` now supports arbirtary transformations using the Transform component:`
+
+![text 2d transforms](text2d_transforms.png)
+
+Note that while `Transform::scale` does have it uses, it is generally still a good idea to adjust text size using the "font size" to ensure it renders "crisply".
+
+## Profiling
+
+### More Spans
+
+<div class="release-feature-authors">authors: @cart, @mockersf, @hymm</div>
+
+The new renderer now has tracing spans for frames, the render app schedule, and the Render Graph (with named Sub Graphs spans). The system executor now has finer grained spans, filling in most of the remaining blanks. Applying System Commands also now has spans.
+
+(ignore those weird characters in the spans ... we're investigating that)
+
+![render app profiling](render_app_profiling.png)
+
+### Tracy Backend
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump)</div>
+
+We now have built in support for the [tracy](https://github.com/wolfpld/tracy) profiler via the `trace_tracy` Cargo feature.
+
+## Transforms
+
+### Friendly Directional Vectors
+
+<div class="release-feature-authors">authors: @guimcaballero</div>
+
+Bevy Transforms now have friendly "directional" functions that return relative vectors:
+
+```rust
+// Points to the left of the transform
+let left: Vec4 = transform.left();
+// Points to the right of the transform
+let right: Vec4 = transform.right();
+// Points up from the transform
+let up: Vec4 = transform.up();
+// Points down from the transform
+let down: Vec4 = transform.down();
+// Points forward from the transform
+let forward: Vec4 = transform.forward();
+// Points back from the transform
+let back: Vec4 = transform.back();
+```
+
+### Transform Builder Methods
+
+<div class="release-feature-authors">authors: @Lythenas</div>
+
+Transforms now have helpful `with_translation()`, `with_rotation()`, and `with_scale()` builder methods:
+
+```rust
+Transform::from_xyz(0.0, 0.0, 10.0).with_scale(Vec3::splat(2.0))
+```
+
+## Rust 2021
+
+<div class="release-feature-authors">authors: @mockersf, @YohDeadfall</div>
+
+Bevy has been updated to use Rust 2021. This means we can take advantage of the new Cargo feature resolver by default (which the new WGPU version requires). Make sure you update your projects to Rust 2021 or you will need to manually enable the new feature resolver!
```

Comment:
Might be a good idea to call out workspaces specifically as a pitfall here for the resolver issue. Could throw in a toml example for emphasis even. It has bitten me a couple of times already when I keep forgetting about it, so no harm in drawing attention to it as a reminder.

Reply:
I'm not sure it's the release note place to explain how rust works... but a link to https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html#details that's explaining what to do could be good